[2023-12-23T16:19:10.132+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: batch_layer.consume_data manual__2023-12-23T16:19:07.048940+00:00 [queued]>
[2023-12-23T16:19:10.137+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: batch_layer.consume_data manual__2023-12-23T16:19:07.048940+00:00 [queued]>
[2023-12-23T16:19:10.138+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2023-12-23T16:19:10.148+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): consume_data> on 2023-12-23 16:19:07.048940+00:00
[2023-12-23T16:19:10.155+0000] {standard_task_runner.py:60} INFO - Started process 4588 to run task
[2023-12-23T16:19:10.157+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'batch_layer', 'consume_data', 'manual__2023-12-23T16:19:07.048940+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/BatchLayerDag.py', '--cfg-path', '/tmp/tmp6oo3orc7']
[2023-12-23T16:19:10.159+0000] {standard_task_runner.py:88} INFO - Job 62: Subtask consume_data
[2023-12-23T16:19:10.182+0000] {task_command.py:423} INFO - Running <TaskInstance: batch_layer.consume_data manual__2023-12-23T16:19:07.048940+00:00 [running]> on host 7de9f6b6c8f6
[2023-12-23T16:19:10.224+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='batch_layer' AIRFLOW_CTX_TASK_ID='consume_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-23T16:19:07.048940+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-23T16:19:07.048940+00:00'
[2023-12-23T16:19:10.267+0000] {hive.py:475} INFO - USE `finnhub_db`
[2023-12-23T16:19:10.301+0000] {hive.py:475} INFO - 
        CREATE TABLE IF NOT EXISTS finnhub_table
        ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.avro.AvroSerDe'
        STORED AS INPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat'
        OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat'
        TBLPROPERTIES ('avro.schema.literal'='{"type": "record", "name": "TradeRecord", "fields": [{"name": "p", "type": "double"}, {"name": "exchange", "type": "string"}, {"name": "crypto_pair", "type": "string"}, {"name": "t", "type": "long"}, {"name": "v", "type": "double"}]}')
    
[2023-12-23T16:19:10.352+0000] {hive.py:475} INFO - USE `finnhub_db`
[2023-12-23T16:19:10.384+0000] {hive.py:475} INFO - 
        CREATE TABLE IF NOT EXISTS temp_avro_data
        ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.avro.AvroSerDe'
        STORED AS INPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat'
        OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat'
        TBLPROPERTIES ('avro.schema.literal'='{"type": "record", "name": "TradeRecord", "fields": [{"name": "p", "type": "double"}, {"name": "exchange", "type": "string"}, {"name": "crypto_pair", "type": "string"}, {"name": "t", "type": "long"}, {"name": "v", "type": "double"}]}')
    
[2023-12-23T16:19:25.280+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-12-23T16:19:25.282+0000] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 4588. PIDs of all processes in the group: [4588]
[2023-12-23T16:19:25.282+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 4588
[2023-12-23T16:20:25.295+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=4588, name='airflow task runner: batch_layer consume_data manual__2023-12-23T16:19:07.048940+00:00 62', status='sleeping', started='16:19:09') did not respond to SIGTERM. Trying SIGKILL
[2023-12-23T16:20:25.296+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGKILL to group 4588
[2023-12-23T16:20:25.301+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=4588, name='airflow task runner: batch_layer consume_data manual__2023-12-23T16:19:07.048940+00:00 62', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='16:19:09') (4588) terminated with exit code Negsignal.SIGKILL
[2023-12-23T16:20:25.301+0000] {standard_task_runner.py:175} ERROR - Job 62 was killed before it finished (likely due to running out of memory)
